rook/migration/v1.11/README.md

```diff
@@ -75,6 +75,14 @@
 
     These will now be provided from here so they must be removed else they will be in conflict.
 
+    ```bash
+    yq4 -i '.rookCeph.monitoring.enabled = false' "$CK8S_CONFIG_PATH/common-config.yaml"
+    yq4 -i '.rookCeph.gatekeeperPsp.enabled = false' "$CK8S_CONFIG_PATH/common-config.yaml"
+    yq4 -i '.networkPolicies.rookCeph.enabled = false' "$CK8S_CONFIG_PATH/common-config.yaml"
```

Comment:
Should we create an issue to remove these from apps?

Reply:
create an issue for this
https://github.com/elastisys/compliantkubernetes-apps/issues/1824